dotCMS/src/integration-test/java/com/dotmarketing/portlets/containers/business/ContainerAPIImplTest.java

```
@@ -328,4 +321,450 @@ private Contentlet unpublish(final String containerInode) throws DotSecurityExce
+        assertEquals(
+                fileAssetContainer.getIdentifier(),
+                fileAssetContainerFromDataBaseWithRelativePath.getIdentifier()
+        );
+
+        assertEquals(
+                fileAssetContainerFromDataBaseWithRelativePath.getHost().getIdentifier(),
+                defaultHost.getIdentifier()
+        );
+    }
+
+
+    /**
+     * Method to Test: {@link ContainerAPIImpl#findContainer(String, User, boolean, boolean)}
+     * When: Find a {@link FileAssetContainer} by relative path and it not exists
+     * Should: return Empty Optional
+     */
+    @Test
+    public void whenFindFileContainerByRelativePathAndNotExists() throws DotSecurityException, DotDataException {
+        final Optional<Container> containerFromDatabase =
+                APILocator.getContainerAPI().findContainer("/not_exists", APILocator.systemUser(), false, false);
+
+        assertFalse(containerFromDatabase.isPresent());
+    }
+
+    /**
+     * Method to Test: {@link ContainerAPIImpl#findContainer(String, User, boolean, boolean)}
+     * When: Find a {@link Container} by Id and it exists but the not have permission
+     * Should: throw a {@link DotSecurityException}
+     */
+    @Test
+    public void whenFindContainerByIdAndUserNotHasPermission() throws DotDataException, DotSecurityException {
+        final User user = new UserDataGen().nextPersisted();
+        final Host host = new SiteDataGen().nextPersisted();
+        final FileAssetContainer fileAssetContainer = new ContainerAsFileDataGen()
+                .host(host)
+                .nextPersisted();
+
+        try {
+            APILocator.getContainerAPI().findContainer(fileAssetContainer.getIdentifier(), user,
+                    false, false);
+            throw new AssertionError("DotSecurityException expected");
+        } catch(DotSecurityException e) {
```

Codacy found an issue: [Avoid empty catch blocks](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)